src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -287,10 +286,23 @@ private void NotifyLeaderAboutRemoval(string dbName)
                 {
                     if (t.Exception != null)
                     {
+                        var message = $"Failed to notify leader about removal of node {_serverStore.NodeTag} from database '{dbName}'";
                         if (_logger.IsInfoEnabled)
                         {
-                            _logger.Info($"Failed to notify leader about removal of node {_serverStore.NodeTag} from database {dbName}", t.Exception);
+                            _logger.Info(message, t.Exception);
                         }
+
+                        var alert = AlertRaised.Create(
+                            null,
+                            message,
+                            $"The database was deleted, but we probably failed to notify the cluster about it.{Environment.NewLine} " +
+                            $"If the node is still shown as pending for deletion, it will require the admin manually to remove the node.",
```

Comment:
How are they going to be able to do that? 
Can we have the cluster observer re-validate this occasionally? Not notifying the cluster observer is actually pretty common in some failure scenarios, so we want a retry on that, and better if the cluster observer does this explicitly.